jetty-hazelcast/src/main/config/modules/session-store-hazelcast-remote.mod

```diff
@@ -14,18 +14,18 @@ sessions
 
 [files]
 maven://com.hazelcast/hazelcast/${hazelcast.version}|lib/hazelcast/hazelcast-${hazelcast.version}.jar
-maven://com.hazelcast/hazelcast-client/${hazelcast.version}|lib/hazelcast/hazelcast-client-${hazelcast.version}.jar
 
 [xml]
 etc/sessions/hazelcast/remote.xml
 
 [lib]
 lib/jetty-hazelcast-${jetty.version}.jar
 lib/hazelcast/hazelcast-${hazelcast.version}.jar
-lib/hazelcast/hazelcast-client-${hazelcast.version}.jar
 
 [ini]
 hazelcast.version?=4.1
+jetty.webapp.addSystemClasses+=,org.eclipse.jetty.hazelcast.
+jetty.webapp.addServerClasses+=,-org.eclipse.jetty.hazelcast.
```

Comment:
I'm not sure this is totally necessary as org.eclipse.jetty package is a server class by default, isn't it?

Reply:
ah yup good catch. not needed for 10.